sw/airborne/modules/computer_vision/lib/v4l/v4l2.c

```diff
@@ -92,8 +97,8 @@ static void *v4l2_capture_thread(void *data)
     assert(buf.index < dev->buffers_cnt);
 
     // Copy the timestamp
-    memcpy(&dev->buffers[buf.index].timestamp, &buf.timestamp, sizeof(struct timeval));
-
+    dev->buffers[buf.index].timestamp = buf.timestamp.tv_usec;
```

Comment:
get_sys_time_usec() also overflows after 72min, but that is normally not a problem...